spylib/utils/constants.py

```diff
@@ -0,0 +1 @@
+ALPHABET = '23456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz'
```

Comment:
Why does the alphabet start at 2?

I'd suggest keeping all the code copied from an external library all in one file or folder

Reply:
I wondered that too, but it's just copied from the code https://github.com/skorokithakis/shortuuid/blob/master/shortuuid/main.py#L42